examples/contracts/examples/contracts/contract.sol

```diff
@@ -6,7 +7,7 @@ contract SimpleStorage {
 
     string _value;
 
-    constructor(string memory value) public {
```

Comment:
Think this is required

Reply:
You are right, this is required until Solidity v0.7.0 [(here)](https://docs.soliditylang.org/en/v0.7.0/070-breaking-changes.html#functions-and-events), and the contract is `>=0.4.24`